controllers/configmap_controller.go

```diff
@@ -83,6 +85,7 @@ type ConfigMapReconciler struct {
 	instanceReconciler
 	servicesManifest *servicescm.Data
 	proxyEnabled     bool
+	VXLANPort        string
```

Comment:
Is there a requirement to export `VXLANPort`?

Reply:
we need to export VXLANPort here in order to use the generateServicesManifest function on line 156
By that point it won't have access to the whole object like it does on 109